packages/server/tests/acceptance/index.spec.ts

```diff
@@ -50,7 +50,8 @@ import { AliasAccount } from '../types/AliasAccount';
 import { setServerTimeout } from '../../src/koaJsonRpc/lib/utils';
 
 chai.use(chaiAsPromised);
-dotenv.config({ path: path.resolve(__dirname, '../../../.env') });
+dotenv.config({ path: path.resolve(__dirname, '../../../../.env') });
+const DOT_ENV = dotenv.parse(fs.readFileSync(path.resolve(__dirname, '../../../../.env')));
```

Comment:
Same here, the change in the line above which modifies the path of the `.env` file is sufficient to fix the problem, fallbacks to this `DOT_ENV` variable is useless, since the same variables are already loaded into `process.env`